### DISCUSSION TITLE
N-Queens in new language

### DISCUSSION THREAD

**Discussion by iArnold** [attractivechaos/plb2#60] [General]
Hi there!
I implemented the N-Queens in a language called ( [Meta](https://language.metaproject.frl/) ) and I wondered if you would be open to add this to your plb2 repository. As Meta does not provide user functions (or methods) the implementation is done using states. The other difference is in the algorithm itself. Instead of just placing the next queen in the next row and check afterwards, the checking is done before the placement, checking for uncovered fields. 
As I am notorious in messing up git repo's, branching and pushing pull requests, I better just tell you where you can find my [working code](https://arnoldvanhofwegen.com/the/unofficial/meta/examples.html).
Best,
Arnold

**Comment by attractivechaos** [comment]
I am happy to add Meta. However, you need to implement the same algorithm. Also, it is recommended to implement matrix multiplication; otherwise Meta won't be shown in the figure.
